.devcontainer/README.md

```diff
@@ -0,0 +1,59 @@
+# Electron Dev on Codespaces
+
+Welcome to the Codespaces Electron Developer Environment.
+
+## Quick Start
+
+Upon creation of your codespace you should have [build tools]() installed and an initialized gclient checkout of Electron.  In order to build electron you'll need to run the following commands.
+
+```bash
+e sync -vv
+e build
+```
+
+The initial sync will take approximately ~30 minutes and the build will take ~8 minutes.  Incremental syncs and incremental builds are substantially quicker.
+
+## Directory Structure
+
+Codespaces doesn't lean very well into gclient based checkouts, the directory structure is slightly strange.  There are two locations for the `electron` checkout that both map to the same files under the hood.
+
+```graphql
+# Primary gclient checkout container
+/workspaces/gclient/* 
+  └─ src/* - # Chromium checkout
+     └─ electron - # Electron checkout
+# Symlinked Electron checkout (identical to the above)
+/workspaces/electron
+```
+
+## Goma
+
+If you are a maintainer [with Goma access]() it should be automatically configured and authenticated when you spin up a new codespaces instance.  You can validate this by checking `e d goma_auth info` or by checking that your build-tools configuration has a goma mode of `cluster`.
```

Comment:
Do we want to add a link here for GOMA access or an explainer of what GOMA is? Sorry, I just see the empty parens and wanted to make sure